### DISCUSSION TITLE
Let the user hide the "organization" sidebar

### DISCUSSION THREAD

**Discussion by awsms** [Kong/insomnia#7069] [Ideas]
As it's posible to hide the sidebar that contains pertinent information (all the requests, ability to manage cookies etc), I'd expect the organization sidebar to be hideable as well, since many of us won't ever use it.

**Comment by subnetmarco** [comment]
This is going to be enabled next week with a control in the bottom footer.

**Before:**

![image](https://github.com/Kong/insomnia/assets/964813/6a3c55f3-f42b-4b78-983b-05bc4ebfe3f8)

**After:**

![image](https://github.com/Kong/insomnia/assets/964813/af42b3e4-fbaf-4056-b8be-a9c8a13251f9)
